package.json

```diff
@@ -71,7 +71,7 @@
     "eslint-config-airbnb": "^16.0.0",
     "eslint-config-edx": "^4.0.1",
     "eslint-import-resolver-webpack": "^0.8.1",
-    "eslint-plugin-compat": "^2.1.0",
+    "eslint-plugin-compat": "^2.2.0",
```

Comment:
Why was this updated?

Reply:
So it looks like [`eslint-plugin-compat` is pinned to `2.2.0` in `package-lock.json`](https://github.com/edx/paragon/blob/master/package-lock.json#L6213)

![image](https://user-images.githubusercontent.com/8136030/37528204-68d5bd78-290a-11e8-9697-a5d87ac96433.png)

So this shouldn't change anything.